src/resources/language/_language-de.yml

```diff
@@ -64,6 +64,10 @@ crossref-lof-title: "Abbildungsverzeichnis"
 crossref-lot-title: "Tabellenverzeichnis"
 crossref-lol-title: "Listingverzeichnis"
 
+environment-proof-title: "Nachweisen"
+environment-remark-title: "Bemerkung"
```

Comment:
I think "Anmerkung" would be a better fit

Reply:
"Bemerkung" is perfect!